scripts/run_kops_test.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# This script creates kOps cluster and runs full integration tests
+set -euoE pipefail
+
+cleanup(){
+
+  exitcode=$?
+  if [[ ${exitcode} -gt 0 ]]; then
+    printf "Error occurred. Attempting to clean setup.. \n"
+  else
+    printf "Tests finished. Attempting to clean setup.. \n"
+  fi
+
+  if ! uninstall_aws_lb_controller; then
+    printf "Uninstalling AWS LB Controller failed.. \n"
+  fi
+
+  if [[ ! -z "${policy_arn}" ]]; then
+    if ! detach_policy_to_role "${policy_arn}" "nodes.${CLUSTER_NAME}"; then
+      printf "Detaching policy from role failed.. \n"
+    fi
+
+    if ! iam::delete_policy "${policy_arn}" "${REGION}"; then
+      printf "Delete policy failed.. \n"
+    fi
+  fi
+
+  if ! delete_kops_cluster; then
+    printf "Delete cluster failed.. \n"
+  fi
+}
+
+: "${TEST_ID:=$RANDOM}"
+: "${REGION:='us-west-2'}"
+: "${KOPS_BINARY_VERSION:='v1.25.3'}"
+: "${KUBERNETES_VERSION:=1.26.0}"
+: "${NODE_SIZE:='c5.xlarge'}"
+: "${NODE_COUNT:=3}"
+: "${OS_OVERRIDE:="linux"}"
+: "${ENDPOINT:=""}"
+: "${SKIP_MAKE_TEST_BINARIES:=false}"
+: "${EXTRA_GINKGO_FLAGS:=''}"
+: "${FOCUS:=''}"
+: "${PRINT_CONTROLLER_LOGS:=false}"
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+GINKGO_TEST_BUILD="$SCRIPT_DIR/../test/build"
+CONTROLLER_IAM_POLICY_FILE="$(dirname "${BASH_SOURCE[0]}")/../docs/install/iam_policy.json"
+CONTROLLER_IAM_POLICY_NAME="aws-lb-controller-kops-${TEST_ID}"
+KOPS_S3_BUCKET="kops-alb-eks-${AWS_ACCOUNT_ID}"
```

Comment:
Is AWS_ACCOUNT_ID always going to be set? If not, you might have to do something like 
  AWS_ACCOUNT_ID=$(aws sts get-caller-identity --region ${REGION} --query Account --output text)